packages/terra-avatar/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "terra-avatar",
+  "main": "lib/Avatar.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-avatar",
+    "Avatar",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-props-table": "^1.10.1"
```

Comment:
If https://github.com/cerner/terra-core/pull/1301 goes into master before this, you'll want to remove terra-props-table from the devDependencies as well as the props-table npm script: props-table

Reply:
Alright, I'll keep an eye on it. I think it'll probably make it in before this does.

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/901237e7a6938c3655ffa65a772ace2bb6d7144e).